### PR TITLE
Add runtime argument for minimum cow transfer threshold

### PR DESCRIPTION
**Pull request by harisang**
This PR adds the option to specify a minimum COW transfer amount, effectively ignoring all transfers below that amount.

After merging this PR and before tagging a new release, I will do a PR in the infrastructure repo to set up these arguments correctly.

### CODE REVIEW

**Comment by harisang** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by bh2smith — APPROVED** [review]
Looks good. Note that you may also want to update the dashboard to exclude min amounts.

File: src/fetch/transfer_file.py [hunk 1]
```diff
@@ -108,18 +108,21 @@ def auto_propose(
...
-            payout_transfers,
-        )
-    )
+    payout_transfers = []
+    for tr in payout_transfers_temp:
+        if tr.token is None:
+            if tr.amount_wei > args.min_transfer_amount_wei:
+                payout_transfers.append(tr)
+        else:
+            if tr.amount_wei > args.min_transfer_amount_cow_atoms:
```
Comment (fhenneke) [inline src/fetch/transfer_file.py]: This reads a bit strange in that it does not refer to wei but is called `amount_wei`.
But renaming to `amount` or `amount_atoms` looks a bit overkill for the change in this PR.

**Review by fhenneke — APPROVED** [review]
LGTM

Running it locally produces the expected result.

We should potentially revisit this change once we support layer 2s. Gas optimizations as in this PR are a bit of an implementation detail of mainnet.

**Review by fhenneke — APPROVED** [review]

**Pull request merged**